support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -398,7 +407,11 @@ object PaperValidation {
 
   }
 
-  def postcodeIsWithinDeliveryArea(postcode: String): Result =
+  def postcodeIsWithinNationalDeliveryArea(postcode: String): Result = {
+    Valid // TODO: determine valid postcodes
```

Comment:
I guess we'll have to check with paper round?  I was wondering myself whether they have a mapping from postcode to providers that we can download to S3 on a schedule or whether we always have to hit their API.

Reply:
I believe we’ll always have to hit their API: we could possibly take a look at caching it our side, I suppose. There’s [another Trello card](https://trello.com/c/r557VThC/65-product-build-support-frontend-backend-validates-paperround-coverage-of-provided-postcode-and-chosen-agent) in the backlog to add this validation.